packages/web-app-files/src/mixins/actions/copy.js

```diff
@@ -54,7 +54,7 @@ export default {
   methods: {
     ...mapActions('Files', ['copySelectedFiles']),
     $_copy_trigger({ resources }) {
-      this.copySelectedFiles({ space: this.space, resources })
+      this.copySelectedFiles({ space: this.space, resources, ...this.$language })
```

Comment:
...this.$language at the end could overwrite previous keys.
I prefer to have it at the beginning, it is also easier to read if we always spread upfront.